pkg/runctl/run_manager_test.go

```diff
@@ -1097,6 +1097,120 @@ func Test_RunManager_Start_CreatesTektonTaskRun(t *testing.T) {
 	assert.Assert(t, result != nil)
 }
 
+func Test_RunManager_addTektonTaskRunParamsForImage(t *testing.T) {
+	t.Parallel()
+	examinee := runManager{
+		pipelineRunsConfig: pipelineRunsConfigStruct{
+			Image:           "defaultImage1",
+			ImagePullPolicy: "defaultPolicy1",
```

Comment:
Extract the values into constants to make it even clearer in expectations that the same string is expected. This also avoids typos or inconsistent future changes.

Reply:
Done